### PR TITLE
feat: Add Requirements and Licensing section

### PR DESCRIPTION
**Pull request by ruzniaievdm**
[2U-117](https://youtrack.raccoongang.com/issue/2U-117)
[edx-platform](https://gitlab.raccoongang.com/2u/studio-mfe/edx-platform/-/merge_requests/8) need to merge too

### CODE REVIEW

**Comment by ruzniaievdm** [comment]
@sendr please merge in separate commits

File: src/schedule-and-details/license-section/utils.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { LICENSE_COMMONS_OPTIONS } from './constants';
+
+const generateLicenseURL = (details) => {
+  const url = 'https://creativecommons.org/licenses/';
```
Comment (sendr) [inline src/schedule-and-details/license-section/utils.js:4]: can we move this to constants as well?

File: src/schedule-and-details/license-section/hooks.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    }
+
+    if (licenseType === LICENSE_TYPE.creativeCommons) {
+      const orderedDetails = Object.entries(licenseDetails)
+        .filter(([, value]) => value)
+        .map(([key]) => LICENSE_COMMONS_OPTIONS[key]);
+
+      const updatedString = `${
+        LICENSE_TYPE.creativeCommons
+      }: ver=4.0 ${orderedDetails.join(' ')}`;
```
Comment (sendr) [inline src/schedule-and-details/license-section/hooks.jsx:34]: can we not hardcode the version?

File: src/schedule-and-details/license-section/utils.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  const detailsString = Object.entries(details)
+    .filter(([, value]) => value)
+    .map(([key]) => LICENSE_COMMONS_OPTIONS[key].toLowerCase())
+    .join('-');
+
+  return `${url}${detailsString}/4.0/`;
```
Comment (sendr) [inline src/schedule-and-details/license-section/utils.js:10]: let's also not hardcode the version, I saw you use it twice

**Review by sendr — APPROVED** [review]

**Pull request merged**